src/charm.py

```diff
@@ -92,6 +98,27 @@ def _on_start(self, _) -> None:
         """Ensure that required software is running."""
         pass
 
+    # =======================
+    #  Helpers
+    # =======================
+
+    def _get_mysql_helpers(self):
```

Comment:
why not have this function as a property?
in such case, it will be possible to use it as `self._mysql.update_mysql_configuration()`

Reply:
changed to a property in [37db8b5](https://github.com/canonical/mysql-operator/pull/6/commits/37db8b57689305a169c643d92613f15416ef8710) whose usage will be further formalized in the future